src/assets/TokenListController.test.ts

```diff
@@ -113,33 +217,82 @@ const sampleSingleChainState = {
       decimals: 18,
       occurrences: 11,
       name: 'Synthetix',
-      iconUrl: 'https://airswap-token-images.s3.amazonaws.com/SNX.png',
+      iconUrl:
+        'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f.png',
+      aggregators: [
+        'Aave',
+        'Bancor',
+        'CMC',
+        'Crypto.com',
+        'CoinGecko',
+        '1inch',
+        'Paraswap',
+        'PMM',
+        'Synthetix',
+        'Zapper',
+        'Zerion',
+        '0x',
+      ],
     },
     '0x514910771af9ca656af840dff83e8264ecf986ca': {
       address: '0x514910771af9ca656af840dff83e8264ecf986ca',
       symbol: 'LINK',
       decimals: 18,
       occurrences: 11,
       name: 'Chainlink',
-      iconUrl: 'https://s3.amazonaws.com/airswap-token-images/LINK.png',
+      iconUrl:
+        'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0x514910771af9ca656af840dff83e8264ecf986ca.png',
+      aggregators: [
+        'Aave',
+        'Bancor',
+        'CMC',
+        'Crypto.com',
+        'CoinGecko',
+        '1inch',
+        'Paraswap',
+        'PMM',
+        'Zapper',
+        'Zerion',
+        '0x',
+      ],
     },
     '0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c': {
       address: '0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c',
       symbol: 'BNT',
       decimals: 18,
       occurrences: 11,
       name: 'Bancor',
-      iconUrl: 'https://s3.amazonaws.com/airswap-token-images/BNT.png',
+      iconUrl:
+        'https://static.metaswap.codefi.network/api/v1/tokenIcons/1/0x1f573d6fb3f13d689ff844b4ce37794d79a7ff1c.png',
+      aggregators: [
+        'Bancor',
+        'CMC',
+        'CoinGecko',
+        '1inch',
+        'Paraswap',
+        'PMM',
+        'Zapper',
+        'Zerion',
+        '0x',
+      ],
     },
   },
   tokensChainsCache: {
     '1': {
       timestamp,
-      data: sampleMainnetTokenList,
+      data: sampleMainnetTokensChainsCache,
     },
   },
 };
 
+const sampleBinanceTokensChainsCache = sampleBinanceTokenList.reduce(
+  (output, current) => {
+    output[current.address] = current;
+    return output;
+  },
+  {} as any,
```

Comment:
Same for this

Reply:
Updated as well. Type is now `TokenListMap`

Reply:
I've renamed `RawToken` to be `TokenListToken` for now since it's a token that comes from the `TokenList`. I didn't want to change or use other `Token` types yet since I wanted to keep the scope small. In the future when we consolidate all of the types, I'd like to remove all but one.